src/tests/cross-browser-compatibility.test.ts

```
@@ -5,7 +5,7 @@
  * across different browsers (Chromium, Firefox, and WebKit).
  */
 
-import { test, expect } from '@playwright/test'
+import { test, expect, Page, ConsoleMessage } from '@playwright/test'
```

ESLint

[error] Imports "Page" and "ConsoleMessage" are only used as type.

(@typescript-eslint/consistent-type-imports)